app/helpers/post_location_helper.rb

```diff
@@ -0,0 +1,17 @@
+module PostLocationHelper
+  def link_to_post(post)
+    if post.type == "ExternalPost"
+      link_to(post.title, post.source_url, target: '_blank')
+    else
+      link_to(post.title, post_path(post))
+    end
+  end
+
+  def show_tags(post)
+    if post.type == "ExternalPost"
+      "<span>question</span>".html_safe
+    else
+      post.tags.map { |tag| "<span> #{tag.title} </span>" }.join.html_safe
```

Comment:
We don't need this helper because we are already rendering the tags with links in the html: https://github.com/RailsGEnthusiasts/RubyGameDev.com/blob/implement-tags-part2/app/views/posts/_tags.html.erb